Makefile

```diff
@@ -153,6 +153,10 @@ test-soft_failure-no-reference:
 test-invalid-syntax:
 	tools/check_invalid_syntax
 
+.PHONY: test-pod-whitespace_rule
+test-code-style:
```

Comment:
this should be part of the target "test-static"

Reply:
Done *this change is ready for review*